internal/plugin/plugin.go

```diff
@@ -91,16 +99,10 @@ func sortByDependencyOrder(original []*Plugin, pluginNames map[string]struct{})
 		for _, plugin := range plugins {
 			foundDeps := true
 			for _, dep := range plugin.Deps {
-				name, _, ok := strings.Cut(dep.Plugin, ":")
+				_, _, ok := strings.Cut(dep.Plugin, ":")
 				if !ok {
 					return nil, fmt.Errorf("invalid plugin dependency: %s", dep)
 				}
-				if _, ok := pluginNames[name]; !ok {
-					// Plugin dependency defined elsewhere - this is ok.
-					// We only calculate dependency order for plugins found within the specified path.
```

Comment:
Removed this so we check dependencies between plugins. For now, everything is expected to be contained in this repo.